integration_tests/build.gradle.kts

```diff
@@ -16,28 +15,12 @@ repositories {
     maven(url = "https://dl.bintray.com/kotlin/kotlinx")
 }
 
-detekt {
-    failFast = true // fail build on any finding
-    input = files("src/main/kotlin", "src/test/kotlin")
-    config = files("../config/detekt.yml")
-    autoCorrect = true //auto format for detekt via klint
```

Comment:
Is it possible to have autocorrect feature enabled in ktlint? (or is it enabled out of the box)

Reply:
Will look into it. But this is one feature we want.

Reply:
@pawelpasterz its done. `gradlew formatKotlin`